### DISCUSSION TITLE
flipper call on iphone

### DISCUSSION THREAD

**Discussion by MrK-World** [aleff-github/my-flipper-shits#8] [General]
Hey I am wondering how this applys to flipper device and will it call on your connected phone or someonelses phone?

**Comment by aleff-github** [comment]
You can use Bluetooth to make these kinds of attacks on particular devices such as iPhones or, in general, in case you do not have physical access to the device.

I.e. https://thecomputernoob.com/2023/02/17/bluetooth-badusb-attacks-flipper-zero/
